doc/source/notebooks/03_node_properties.ipynb

```diff
@@ -481,35 +493,37 @@
W/UbVcpLXE/YtMnnziYJhlLSAnPQR7pypTxE8vzzqxf6CgXeJ7dsqb4X+bzPGUwLw6p2E2F/8N5TFpsI8d4TyX/QuKH+Uyj4z42+IyVJL1Yxg462EQWn6rrBaxWLjF13XXR5Jh1fcDAKq0E+ZIt/n21YllUA8H8BXAjg3wF8H8BaxtiPZfufccYZ7Iknnmhcg8bH+VSlUuFToKVL0wsap3PrnnPnTmDDBj6lmp7m061zz+XscEuWAD/3PD1LlvC2Tk0Be/bwhJRTTqk93wMPADfcwKdx09OcGWvTpup2/frXPJj+rLOARYvk9+LZZ/k+L30p8F//xbkDHCf6d42Pc1fMiy8Cxx4LLFvG99+5kyfb0O+87z7g8str75XYzv/7f4EvfhE47jjgmmvkbQX073fw2Tz7LJ/Wlkr8fpfL/N6OjwO33ebfwwcf5DwPYW0NXlt1v085xf8u+Czp3gH+fRPvqWz73r3Ad78LfOhDPq/JddcBb3oT35fuT6Xi9yXxHKo2B++rzneq8RQ1JoLf64yhuOOMjpHdx5iwLOtJxtgZ0u/aQSEDgGVZbwRwL3jExWcZY3eo9m24Qs4aknSuZp4vLWS1XTK0U1uB9mtvG2NBKOQ4OOoUsoGBQdsgTCG3S5SFgYGBwYKHUcgGBgYGGYFRyAYGBgYZgVHIBgYGBhmBUcgGBgYGGYFRyAYGBgYZgVHIBgYGBhmBUcgGBgYGGYFRyAYGBgYZgVHIBgYGBhnBgkydtixrHMC+VrdDwCIAv251I2LCtLk5MG1uPLLW3pMYY1LCkAWpkLMGy7KeUOWuZxWmzc2BaXPj0U7tNS4LAwMDg4zAKGQDAwODjMAo5Obgb1rdgAQwbW4OTJsbj7Zpr/EhGxgYGGQExkI2MDAwyAiMQk4BlmUtsSzrccuyRi3L+rFlWTd4219iWdajlmX9xPt7nLfdsixrq2VZeyzLesayrOVNbm/RsqwRy7J+4LX3w972V1iW9T2vXYOWZTnedtf7f4/3/dJmtjfQ9rxlWbsty/pWO7TZsqy9lmX90LKspy3LesLblsl+IbT5WMuyvmJZ1r9ZlvWsZVmvyXKbLcs62bu/JL+xLOs9WW6zCkYhp4MZADcxxk4FcA6Ad1mWdSqAXgCPMcb+J4DHvP8B4E8B/E9PrgXw6Sa3dxLAGxhjfwTgNAB/YlnWOQA+DuAextirALwAoNvbvxvAC972e7z9WoUbADwr/N8Obb6AMXaaEHqV1X5BuA/APzDGfg/AH4Hf78y2mTH2nHd/TwNwOoDDAP4uy21WQlWO2khyAfANABcBeA7ACd62EwA8531+AMAaYf/5/VrQ1g4ATwE4Gzx4vuBtfw2AIe/zEIDXeJ8L3n5WC9r6cvCB9QYA3wJgtUGb9wJYFNiW2X4B4BgAzwfvVZbbHGjnSgD/0k5tFsVYyCnDmxovA/A9AMczxn7pffWfAI73Pr8MwM+Fw37hbWsavKn/0wDGADwK4KcAXmSMzUjaNN9e7/v9AF7azPZ6uBfA+wHMef+/FNlvMwPwHcuynrQs61pvW2b7BYBXABgH8DnPNfQZy7LKyHabRbwNwE7vc7u0eR5GIacIy7IqAL4K4D2Msd+I3zH+Ks5MSAtjbJbxKd7LAZwF4Pda26JwWJb1ZgBjjLEnW92WmDiXMbYcfJr8LsuyXid+mbV+AT6bWA7g04yxZQAOwZ/qA8hkmwEA3vrBKgBfDn6X1TYHYRRySrAsywZXxl9kjH3N2/wry7JO8L4/AdwaBYB/B7BEOPzl3ramgzH2IoDHwaf7x1qWVZC0ab693vfHAPiv5rYUfwxglWVZewF8CdxtcR+y3WYwxv7d+zsG7tc8C9nuF78A8AvG2Pe8/78CrqCz3GbCnwJ4ijH2K+//dmhzFYxCTgGWZVkAHgTwLGPsbuGrbwJY731eD+5bpu3rvNXecwDsF6ZWzWjvYsuyjvU+l8D93c+CK+a3KtpLv+OtAP7RsziaBsbYBxhjL2eMLQWflv4jY+xqZLjNlmWVLcvqpM/g/s0fIaP9AgAYY/8J4OeWZZ3sbboQwGiW2yxgDXx3BdAeba5Gq53YC0EAnAs+HXoGwNOevBHcZ/kYgJ8A2AXgJd7+FoD7wf22PwRwRpPb+4cAdnvt/RGA27ztrwQwAmAP+LTP9bYXvf/3eN+/ssX3+3wA38p6m722/cCTHwP4oLc9k/1CaPdpAJ7w+sfXARzXBm0ug8+AjhG2ZbrNMjGZegYGBgYZgXFZGBgYGGQERiEbGBgYZARGIRsYGBhkBEYhGxgYGGQERiEbGBgYZARGIRsYeLAsq2RZ1ncty8qncK7FlmX9QxrtMjh6YBSygYGPDQC+xhibrfdEjLFxAL+0LOuP62+WwdECo5ANFjwsyzrT470tetlzP7Ys6w8ku14NL5vLsqzzLY9z2fv/U5Zlvd37vNeyrI8Sx7FlWcstyxqyLOunlmVdJ5zv6945DQy0UIjexcCgvcEY+75lWd8EcDuAEoCHGWM/EvfxiGleyRjbq3nanzHGTrMs6x4A28G5NorgmY8D3j5PeNc0MNCCUcgGRws+AuD7ACYAbJZ8vwjAizHO903v7w8BVBhjBwAcsCxr0rKsYxknbRoD8NuJW2xw1MG4LAyOFrwUQAVAJ7glG8QRyXZL+GwHvpv0/s4Jn+l/MnSK3nkNDLRgFLLB0YIHAPwFgC9CUs6JMfYCgLxlWaJS/gMv8sIFpyeNG33xu+AuDAMDLRiFbLDgYVnWOgDTjLEdAD4G4EzLst4g2fU74Mx9hBfAS0H9Kzhb2Ac8Gk1dXADgfyVrtcHRCMP2ZmDgwas+fCNj7M8syzofwM2MsTfXcb7/DeBSz/o2MIiEsZANDDwwxp4C8HhaiSEA7jbK2CAOjIVsYGBgkBEYC9nAwMAgIzAK2cDAwCAjMArZwMDAICMwCtnAwMAgIzAK2cDAwCAjMArZwMDAICP4/wF7b5YiC9McPgAAAABJRU5ErkJggg==\n",
```

Comment:
If I'm not wrong, this image isn't present in the modified notebook

Reply:
Yeah, sometimes when I run it on a local (well, inside bb5) jupyter instance, it skips a cell. this is what happened here. Should be fixed now